shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -65,9 +99,32 @@ public void setCanAuthenticate(boolean flag) {
     biometricServiceConnected = flag;
   }
 
-  @Implementation(minSdk = R)
-  protected int canAuthenticate(int userId, int authenticators) {
-    return biometricServiceConnected ? BIOMETRIC_SUCCESS : BIOMETRIC_ERROR_NO_HARDWARE;
+  /**
+   * @param num to set the authenticate result
```

Comment:
Could you provide more comments about those `set*` methods? It's better if you can clarify how the set value will affect.

Reply:
sure